drivers/source/serial_io.c

```diff
@@ -24,7 +24,9 @@ void sciMutexInit(void) {
     }
```

Comment:
There should be some modified files in `hal` once you complete the HALCoGen steps in the challenge

Reply:
Oh, I just didn't commit these files.